.github/workflows/node.yml

```diff
@@ -0,0 +1,72 @@
+# Copyright (c) 2024 Hemi Labs, Inc.
+# Use of this source code is governed by the MIT License,
+# which can be found in the LICENSE file.
+
+# GitHub Actions workflow to lint, build and test NPM packages.
+name: "Node"
+on:
+  push:
+    branches: [ "main" ]
+  pull_request:
+    branches: [ "main" ]
+  workflow_dispatch:
+  workflow_call:
+
+concurrency:
+  group: "npm-${{ github.workflow }}-${{ github.event.number || github.ref }}"
+  cancel-in-progress: "${{ github.event_name == 'pull_request' }}"
+
+env:
+  GO_VERSION: "1.22.x"
+  PNPM_VERSION: "9.4.x"
+
+jobs:
+  build:
+    name: "Build"
+    runs-on: "ubuntu-latest"
+    permissions:
+      contents: read
+    steps:
+      - name: "Checkout repository"
+        uses: actions/checkout@v4
+
+      - name: "Setup Go ${{ env.GO_VERSION }}"
+        uses: actions/setup-go@v5
+        with:
+          go-version: "${{ env.GO_VERSION }}"
+          cache: true
+          check-latest: true
+
+      - name: "Setup pnpm ${{ env.PNPM_VERSION }}"
+        uses: pnpm/action-setup@v4
+        with:
+          version: "${{ env.PNPM_VERSION }}"
+
+      - name: "Setup Node ${{ env.NODE_VERSION }}"
```

Comment:
`${{ env.NODE_VERSION }}` doesn't exist anymore!

```suggestion
      - name: "Setup Node"
```


(Same in `.github/workflows/release.yml`)

Reply:
Again, not a blocker as actions still work, and an empty space is logged

<img width="325" alt="image" src="https://github.com/hemilabs/heminetwork/assets/352474/14af521d-43eb-471a-8be0-6e0f600bf7ac">


So feel free to merge either way if you prefer it that way 😄

Reply:
Good catch! Thanks!